### ISSUE TITLE
Provide better error handling around missing routing directory

### ISSUE THREAD

**Issue by designfrontier** [ansble/monument#440]
```
{ Error: ENOENT: no such file or directory, scandir 'somewhere'
at Error (native)
at Object.fs.readdirSync (fs.js:953:18)
at setup (/home/rof/src/github.com/ansble/monument/routes/serverSetup.js:9:565)
at assert.throws (/home/rof/src/github.com/ansble/monument/routes/serverSetup_test.js:9:835)
at Proxy.assertThrows (/home/rof/src/github.com/ansble/monument/node_modules/chai/lib/chai/core/assertions.js:2554:7)
at Proxy.methodWrapper (/home/rof/src/github.com/ansble/monument/node_modules/chai/lib/chai/utils/addMethod.js:57:25)
at Function.assert.throws (/home/rof/src/github.com/ansble/monument/node_modules/chai/lib/chai/interface/assert.js:1973:16)
at Context.it (/home/rof/src/github.com/ansble/monument/routes/serverSetup_test.js:9:784)
at callFn (/home/rof/src/github.com/ansble/monument/node_modules/mocha/lib/runnable.js:354:21)
at Test.Runnable.run (/home/rof/src/github.com/ansble/monument/node_modules/mocha/lib/runnable.js:346:7)
at Runner.runTest (/home/rof/src/github.com/ansble/monument/node_modules/mocha/lib/runner.js:442:10)
at /home/rof/src/github.com/ansble/monument/node_modules/mocha/lib/runner.js:560:12
at next (/home/rof/src/github.com/ansble/monument/node_modules/mocha/lib/runner.js:356:14)
at /home/rof/src/github.com/ansble/monument/node_modules/mocha/lib/runner.js:366:7
at next (/home/rof/src/github.com/ansble/monument/node_modules/mocha/lib/runner.js:290:14)
at Immediate.<anonymous> (/home/rof/src/github.com/ansble/monument/node_modules/mocha/lib/runner.js:334:5)
at runCallback (timers.js:672:20)
at tryOnImmediate (timers.js:645:5)
at processImmediate [as _immediateCallback] (timers.js:617:5)
errno: -2,
code: 'ENOENT',
syscall: 'scandir',
path: 'somewhere' } 'somewhere'
```

Catch this error and provide a more descriptive error message explaining what is wrong and how to correct it.

**Comment by dottorblaster** [comment]
I'd like to work on this too as I'm getting more and more confident with the codebase over time.

**Comment by designfrontier** [comment]
This one was closed by #443
